### ISSUE TITLE
[no-invalid-this] TypeError: rules.FunctionExpression is not a function

### ISSUE THREAD

**Issue by TomMettam** [typescript-eslint/typescript-eslint#4524]
- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

**Repro**

```JSON
{
  "root": true,
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "tsconfig.json"
  },
  "plugins": [
    "@typescript-eslint"
  ],
  "extends": [
    "plugin:@typescript-eslint/all"
  ]
}
```

```TS
export class Test
{
    public someFunction()
    {
        
    }
}
```

**Expected Result**

No error

**Actual Result**

TypeError: rules.FunctionExpression is not a function

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.10.2` |
| `@typescript-eslint/parser`        | `5.10.2` |
| `TypeScript`                       | `4.5.4` |
| `ESLint`                           | `8.8.0` |
| `node`                             | `14.17.6` |

**Comment by TomMettam** [comment]
Sorry, I should mention: disabling `no-invalid-this` clears the error

**Comment by bradzacher** [comment]
This was fixed (#4444)
If you're still facing it - you likely have multiple versions of our plugin installed (see the FAQ article).

**Comment by TomMettam** [comment]
Hello, what version was this included in?
